docs/roadmap.md

```
@@ -0,0 +1,60 @@
+    - [ ] **metadata**
+    - [ ] **modules**
+    - [ ] **show**
+    - [ ] **refresh**
+    - [ ] **taint/untaint**
+    - [ ] **show**
+    - [ ] **workspace**
+    - [ ] **state**
+
+- ✅ **Commands history** - TerraLand will keep track of all the commands you run, provide a history of your actions and allow you to re-run them.
+- [ ] **Workspaces** - TerraLand will allow you to manage multiple workspaces, making it easier to manage different environments.
+    - [ ] **Create**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)